mne/viz/evoked.py

```diff
@@ -875,3 +880,158 @@ def plot_snr_estimate(evoked, inv, show=True):
     plt.draw()
     plt_show(show)
     return fig
+
+
+def _connection_line(x, fig, sourceax, targetax):
+    """Helper function to connect time series and topolots"""
+    from matplotlib.lines import Line2D
+    transFigure = fig.transFigure.inverted()
+    tf = fig.transFigure
+
+    (xt, yt) = transFigure.transform(targetax.transAxes.transform([.5, 0]))
+    (xs, _) = transFigure.transform(sourceax.transData.transform([x, 0]))
+    (_, ys) = transFigure.transform(sourceax.transAxes.transform([0, 1]))
+    return Line2D((xt, xs), (yt, ys), transform=tf, color='grey',
+                  linestyle='-', linewidth=1.5, alpha=.66, zorder=-1)
+
+
+def joint_plot(evoked, times="peaks", title='', picks=None, exclude=list(),
+               show=True, ts_args=dict(), topomap_args=dict()):
```

Comment:
do not use mutable default arguments. So `exclude=(), ts_args=None, topomap_args=None` and later have e.g. `topomap_args = dict() if topomap_args is None else topomap_args`